### ISSUE TITLE
websocket error

### ISSUE THREAD

**Issue by MansoorJafari9** [flutter-webrtc/dart-sip-ua#222]
Hi
I dont have websocket for set to webSocketUrl 
Can I set this parameter to null?

**Comment by cloudwebrtc** [comment]
No, this package only supports sip over WebSockets, so webSocketUrl is necessary.

**Comment by MansoorJafari9** [comment]
Thanks for your answer.
excuse me, What permissions should be taken from user in android?
